### PR TITLE
Sprint3 29 nav list html

### LINKED ISSUE #29: The TAMU Footer and the Header's top nav should provide all links statically
These should be expressed within maps in the ts components, and displayed in the templates via an ngFor loop.

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #29

### CODE REVIEW

File: src/app/tl-footer/tl-footer.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <wvr-footer-element [background]="backgroundColor">
   <wvr-nav-list-element aligned="CENTER">
-    <ng-container wvr-nav-items>
-      <ng-content select="wvr-nav-li"></ng-content>
-    </ng-container>
+    <wvr-nav-li-element *ngFor="let link of links; let i = index" href="{{link.href}}">
```
Comment (jeremythuff) [inline src/app/tl-footer/tl-footer.component.html]: Is the let `i = index` needed?

File: src/app/tl-header/tl-header.component.html [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
   <wvr-nav-list-element top-navigation aligned="RIGHT">
-    <wvr-nav-li-element href="#action">
-      <wvr-text value="Action"></wvr-text>
-    </wvr-nav-li-element>
-    <wvr-nav-li-element href="#link">
-      <wvr-text value="Link"></wvr-text>
-    </wvr-nav-li-element>
-    <wvr-nav-li-element href="#help">
-      <wvr-text value="Help"></wvr-text>
+    <wvr-nav-li-element *ngFor="let link of toplinks; let i = index" href="{{link.href}}">
```
Comment (jeremythuff) [inline src/app/tl-header/tl-header.component.html]: See comment above...

File: src/app/tl-footer/tl-footer.component.html (hunk 1 above)
Comment (rmathew1011) [reply src/app/tl-footer/tl-footer.component.html]: Removed this expression

File: src/app/tl-header/tl-header.component.html (hunk 2 above)
Comment (rmathew1011) [reply src/app/tl-header/tl-header.component.html]: Resolved  - removed the expression

**Pull request merged**